lib/GridListImageItem/GridListImageItem.js

```diff
@@ -191,6 +191,7 @@ module.exports = kind(
 	* @private
 	*/
 	imageSizingChanged: function () {
+		this.$.image.addRemoveClass('enyo-gridlist-imageitem-image', (!this.imageSizing && !!this.placeholder));
```

Comment:
Hi @aarontam I didn't see 'sized-image' class was being applied here at that time. I thought it's 'enyo.Image' related class. So i wanted the affect to be minimal and confined.
One query: !!this.placeholder is also not required? If there is no placeholder letting the image to take a natural size ?